Mathlib/Analysis/LocallyConvex/Bounded.lean

```diff
@@ -102,6 +102,38 @@ theorem IsVonNBounded.union {s₁ s₂ : Set E} (hs₁ : IsVonNBounded 𝕜 s₁
 
 end Zero
 
+section ContinuousAdd
+
+variable [SeminormedRing 𝕜] [AddZeroClass E] [TopologicalSpace E] [ContinuousAdd E]
+  [DistribSMul 𝕜 E] {s t : Set E}
+
+protected theorem IsVonNBounded.add (hs : IsVonNBounded 𝕜 s) (ht : IsVonNBounded 𝕜 t) :
+    IsVonNBounded 𝕜 (s + t) := fun U hU ↦ by
```

Comment:
Is the iff not true? ie
```suggestion
lemma isVonNBounded_add : IsVonNBounded 𝕜 (s + t) ↔ IsVonNBounded 𝕜 s ∧ IsVonNBounded 𝕜 t := sorry
```

Reply:
Not true if `s` is empty and `t` is unbounded.

Reply:
Sorry, I meant to ask the version where both `s` and `t` are nonempty